app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -130,7 +130,6 @@ public void shouldShowEventError() {
         inOrder.verify(eventDetailView).showProgressBar(true);
         inOrder.verify(eventRepository).getEvent(id, false);
         inOrder.verify(eventDetailView).showEventLoadError(error);
-        inOrder.verify(eventDetailView).showProgressBar(false);
```

Comment:
Test should test app's behavior not what developer has coded. I will add test to check if progressbar gets hidden when complete data gets loaded. :+1: